main.go

```diff
@@ -34,12 +41,14 @@ func main() {
 	// Load keys for JWT verification
 	if config.Server.jwtpubkeyurl != "" {
 		if err := auth.getjwtpubkey(config.Server.jwtpubkeyurl); err != nil {
-			log.Panicf("Error while getting key %s: %v", config.Server.jwtpubkeyurl, err)
+			log.Panicf("Error while getting key %s: %v",
+				config.Server.jwtpubkeyurl, err)
 		}
 	}
 	if config.Server.jwtpubkeypath != "" {
 		if err := auth.getjwtkey(config.Server.jwtpubkeypath); err != nil {
-			log.Panicf("Error while getting key %s: %v", config.Server.jwtpubkeypath, err)
+			log.Panicf("Error while getting key %s: %v",
+				config.Server.jwtpubkeypath, err)
```

Comment:
why the line break?

Reply:
just a habit to limit lines to 80 characters! I'll revert it :)